client/package.json

```diff
@@ -5,7 +5,9 @@
   "dependencies": {
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "react-scripts": "3.1.1"
+    "react-scripts": "3.1.1",
+    "semantic-ui-css": "^2.4.1",
```

Comment:
why semantic ui?

Reply:
Because it has [Placeholders](https://semantic-ui.com/elements/placeholder.html). Which could be useful for an app like ours.
However, we just found out that the modals are not accessible. Hence reconsidering this decision.